source/guide_nextcloud.rst

```diff
@@ -69,22 +124,65 @@ Installation
   100 37.5M  100 37.5M    0     0  5274k      0  0:00:07  0:00:07 --:--:-- 6062k
  [isabell@stardust html]$
 
-Now point your browser to your uberspace URL and follow the instructions.
-
+Setup
+-----
 .. warning:: We strongly recommend you to use the MySQL backend for nextcloud. Do not use the SQLite backend for production. It gives you a better performance and reduces disk load on the host you share.
 
-You will need to enter the following information:
+If you are not comfortable with the the fist step via command line you can also go the web way by pointing your browser to https://isabell.uber.space
```

Comment:
I am not sure about keeping this in. Users (mistakenly) configuring nextcloud to use sqlite is a bigger problem for us than it seems. They increase the system and IO load significantly. To get to this point in the instructions, you already managed to download nextcloud via the CLI, edit files, and restart php. Executing a few more `php occ` lines seems fair game to me.

What do you think?

Reply:
Okay, do you think we should keep the `.. warning::` part?
I think we can remove the whole section under the headline "Setup" until "command line based installation"